packages/manager/modules/netapp/src/dashboard/index/template.html

```diff
@@ -74,6 +74,10 @@
         on-error="$ctrl.onBillingInformationError(error)"
         service-path="{{ ::'/storage/netapp/' + $ctrl.storage.id }}"
         with-engagement="$ctrl.isCommitmentAvailable"
+        tracking-prefix="netapp::dashboard"
+        commit-impression-data="$ctrl.shouldReengage() ?
```

Comment:
Could this test be made in a separate controller method? Also, can it be applied once? (Method call prefixed by `::`, or even a variable to store the result)

Reply:
It's applied